src/pudl/metadata/fields.py

```diff
@@ -4012,6 +4032,63 @@
         "type": "string",
         "description": "Label describing types of revenues.",
     },
+    "revenue_requirement_category": {
+        "type": "string",
+        "description": (
+            "A category of revenue requirement components. "
```

Comment:
nit: the way "category" and "component" are used here suggests nested categories and components that *I think* isn't what we're trying to convey. Maybe say: "categorized components of utility revenue requirements"?

Reply:
"The category of revenue requirement associated with each component of utility's income statements. "?